src/UploadHandler.cpp

```diff
@@ -151,13 +148,11 @@ void UploadHandler::handleRequest(HTTPRequest &request, HTTPResponse &response)
 {
 	if (!checkFiles(request))
 	{
-		std::cout << PURPLE << "calling handle response" << RESET << std::endl;
 		handleResponse(response, BAD_REQUEST);
 		return;
 	}
 	if (!request.getUploadBoundary().empty())
 	{
-		std::cout << PURPLE << "calling upload boundary" << RESET << std::endl;
 		if (!createFile(const_cast<HTTPRequest &>(request)))
 			return (response.setStatusCode(422, "Unprocessable Entity"));
```

Comment:
Why remove these two errors log?

Reply:
It is not necessary because handleResponse and createFIle have prints inside